### ISSUE TITLE
Title of the paseos start page should be the name/title of the first slide?

### ISSUE THREAD

**Issue by earseneau** [google/paseos#10]
This would make it such that if I had multiple journeys I could search for them more easily in the Chrome URL box.

**Comment by diegowilson** [comment]
Do you mean the HTML page title that shows up in the tab name?

**Comment by earseneau** [comment]
Changed title of bug to make it clear it was the HTML Page Title

**Comment by diegowilson** [comment]
Yep, I totally agree with this one. I'll probably fix this in my next round of updates.
